pyanaconda/bootloader.py

```diff
@@ -2257,6 +2311,9 @@ def boot_dir(self):
         return "/boot"
 
     def write_config_images(self, config):
+        if flags.cmdline.getbool("blscfg", True):
```

Comment:
Please do not use this string directly everywhere. The ideal solution would be to add this as bool flag to the `flags` class. However creating constant and moving it to constants would be enough.

Reply:
Oh, I'd already done that and just didn't realize it would translate to flags.blscfg ; cool, will fix.